### DISCUSSION TITLE
How to realize FSI in dafoam, shape optimization of models with elastic constraints

### DISCUSSION THREAD

**Discussion by Pony1235** [mdolab/dafoam#507] [General]
How to increase the elastic constraints in the model to optimize the target as the displacement such as Flow-induced vibration, or forced vibration? Which modules should I modify? Thanks a lot.

**Comment by friedenhe** [comment]
Are you talking about vibration for unsteady FSI? DAFoam does not support unsteady optimization yet (we are working on it but it may take quite a while until it is ready)...
